src/View/Exception/MissingCellViewException.php

```diff
@@ -13,15 +13,48 @@
  */
 namespace Cake\View\Exception;
 
-use Cake\Core\Exception\Exception;
-
 /**
  * Used when a view file for a cell cannot be found.
  */
-class MissingCellViewException extends Exception
+class MissingCellViewException extends MissingTemplateException
```

Comment:
Nitpick: Perhaps the class name could be changed to `MissingCellTemplateException`.